pkg/ctl/utils/describe_stacks.go

```diff
@@ -1,26 +1,38 @@
 package utils
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 	"github.com/kris-nova/logger"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/printers"
 
 	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
-	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
 )
 
 func describeStacksCmd(cmd *cmdutils.Cmd) {
 	cfg := api.NewClusterConfig()
 	cmd.ClusterConfig = cfg
 
 	var all, events, trail bool
+	var output string
 
 	cmd.SetDescription("describe-stacks", "Describe CloudFormation stack for a given cluster", "")
 
 	cmd.CobraCommand.RunE = func(_ *cobra.Command, args []string) error {
 		cmd.NameArg = cmdutils.GetNameArg(args)
-		return doDescribeStacksCmd(cmd, all, events, trail)
+		switch output {
+		case "":
+			return doDescribeStacksCmd(cmd, all, events, trail, false)
+		case "json":
+			return doDescribeStacksCmd(cmd, all, events, trail, true)
+		default:
+			return fmt.Errorf("unknown output format: %s", output)
+		}
```

Comment:
It should be straightforward to add support for YAML as well (the `table` output type doesn't make sense here).